src/Simulation/TargetDefinitions/TargetPackages/Type1.Package.props

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
A very short readme in the TargetDefinitions folder would be useful such that people are not confused by the expectation that all files in the Decomposition and/or Intrinsic folder should belong to the same package.

Reply:
Agreed. There is an outstanding checkbox for that in #249, but I'm torn on whether I want to include that as part of this PR or in a follow up. It's probably worth the effort to do it now so that folks can find their way around the rearranged code right away!

Reply:
Added a README in https://github.com/microsoft/qsharp-runtime/pull/421/commits/ef53ae73f4d684b0c32de074f992f370ef2e8fc4